### PR TITLE
=doc Transform broken link to ActorNotFound to inline code #23517

### PR DESCRIPTION
**Pull request by jlprat**
Refs: #23517
Broken link by mistake, should just be the class name surrounded by backticks

### CODE REVIEW

**Comment by jlprat** [comment]
I could also convert this to a ScalaDoc / JavaDoc link, but I think it would be the only place where Akka Docs link to them.

File: akka-docs/src/main/paradox/scala/actors.md [hunk 1]
```diff
@@ -610,7 +610,7 @@ You can also acquire an `ActorRef` for an `ActorSelection` with
 the `resolveOne` method of the `ActorSelection`. It returns a `Future`
 of the matching `ActorRef` if such an actor exists. @java[(see also
 @ref:[Java 8 Compatibility](java8-compat.md) for Java compatibility).] It is completed with
-failure [[akka.actor.ActorNotFound]] if no such actor exists or the identification
+failure `ActorNotFound` if no such actor exists or the identification
```
Comment (ktoso) [inline akka-docs/src/main/paradox/scala/actors.md]: Leave as `akka.actor.ActorNotFound`?

Comment (jlprat) [inline akka-docs/src/main/paradox/scala/actors.md]: Yes, also an option

Comment (jlprat) [inline akka-docs/src/main/paradox/scala/actors.md]: Changed!

**Review by patriknw — APPROVED** [review]
LGTM

**Comment by patriknw** [comment]
thanks

**Pull request merged**